template/.gitignore

```diff
@@ -11,3 +11,5 @@ logs
 
 # dependencies
 node_modules
+
+package-lock.json
```

Comment:
So it's actually standard to commit package-lock files, so this should not be ignored

Reply:
you mean using exact version in `package.json`? sure